hooks/use-onboarding-wizard.tsx

```
@@ -0,0 +1,202 @@
+    const resetWizard = useCallback(() => {
+        setState(INITIAL_STATE);
+    }, []);
+
+    const canProceedToNext = useCallback(() => {
+        switch (state.step) {
+        case 0: // Client selection
+            return !!state.client;
+        case 1: // Goals setup
+            return state.goals.stepsGoal > 0 && state.goals.waterGoal > 0 && state.goals.sleepGoal > 0;
+        case 2: // Workout plan & schedule
+            const hasPlan = state.program.createNewPlan || state.program.workoutPlanId;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)